_artifacts/document-attributes.adoc

```diff
@@ -217,7 +217,7 @@ ifdef::PAM[]
 :DEPLOYING_MANAGED_FREEFORM_ON_OPENSHIFT: Deploying a {PRODUCT} freeform managed server environment on {OPENSHIFT}
 :URL_DEPLOYING_AUTHORING_ON_OPENSHIFT: {URL_BASE_ENTERPRISE}/deploying_a_{URL_COMPONENT_PRODUCT}_authoring_environment_on_red_hat_openshift_container_platform
 :URL_DEPLOYING_MANAGED_FIXED_ON_OPENSHIFT: {URL_BASE_ENTERPRISE}/deploying_a_{URL_COMPONENT_PRODUCT}_managed_fixed_server_environment_on_red_hat_openshift_container_platform
-:URL_DEPLOYING_MANAGED_FREEFORM_ON_OPENSHIFT: {URL_BASE_ENTERPRISE}/deploying_a_{URL_COMPONENT_PRODUCT}_managed_freeform_server_environment_on_red_hat_openshift_container_platform
+:URL_DEPLOYING_MANAGED_FREEFORM_ON_OPENSHIFT: {URL_BASE_ENTERPRISE}/deploying_a_{URL_COMPONENT_PRODUCT}_freeform_managed_server_environment_on_red_hat_openshift_container_platform
```

Comment:
@csherrar , Have you cross verified this URL variable with Misha?

Reply:
Yup, sent him the link via gchat